cmd/kubeadm/app/cmd/init.go

```diff
@@ -114,12 +115,14 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 	var featureGatesString string
 	var criSocket string
 	var ignorePreflightErrors []string
+	var verbosityLevel int
```

Comment:
do you really need that variable now ?

Reply:
Nope. I wonder how me and test scripts missed it. I'll remove it.